services/storage-users/pkg/command/uploads.go

```diff
@@ -31,7 +36,7 @@ func Uploads(cfg *config.Config) *cli.Command {
 func ListUploads(cfg *config.Config) *cli.Command {
 	return &cli.Command{
 		Name:  "list",
-		Usage: "Print a list of all incomplete uploads",
+		Usage: "Print a list of all incomplete uploads (deprecated, use sessions)",
```

Comment:
Maybe also print the deprecation warning when the command is executed? Better to be explicit than sorry.

Reply:
I don't want to risk interfering with existing scripts that try to parse the output.

Reply:
Then only print it when output is not set to `json`